calico_mesos/calico_mesos.py

```diff
@@ -71,6 +72,19 @@ def calico_mesos():
     except KeyError:
         raise IsolatorException(ERROR_MISSING_ARGS)
 
+    # Unfortunately, netmodules passes in labels poorly, so move them
+    # around to match the agreed upon design.
+    if args.has_key('labels'):
+        # copy labels so we're not modifiying the original as we iterate
+        _labels = copy.deepcopy(args['labels'])
+        args['labels'] = {}
```

Comment:
The docstrings for `isolate`, `reserve`, and `allocate` all show the expected format. Some arguments are different from each, so it doesn't make sense to document it here. They _do_ all show the labels field, so I suppose I can document its before and after, here.